state/src/item/asset.rs

```diff
@@ -26,7 +26,29 @@ pub struct Asset {
     amount: u64,
 }
 
+impl Encodable for Asset {
```

Comment:
Use `RlpEncodable` and `RlpDecodable` instead of implementing it manually.

Reply:
And please add a test case for it.

Reply:
I resolved it.